cmake/modules/BuildDPDK.cmake

```diff
@@ -95,8 +95,13 @@ function(do_build_dpdk dpdk_dir)
     BUILD_COMMAND ${make_cmd} O=${dpdk_dir} CC=${CMAKE_C_COMPILER} EXTRA_CFLAGS=-fPIC
     BUILD_IN_SOURCE 1
     INSTALL_COMMAND "true")
+  if(numa_FOUND)
```

Comment:
could you move `find_package(numa QUIET)` here ? to where `numa_FOUND` is used the first time.

Reply:
The result of find_package (numa QUIET)  is used in both do_build_dpdk and do_export_dpdk. Put find_package (numa QUIET) in do_build_dpdk, NUMA_FOUND is true in do_build_dpdk but false in do_export_dpdk.